app/views/mountain_view/styleguide/index.html.erb

```diff
@@ -1,22 +1,22 @@
 <div class="mv-main__header">
-  <h1>Styleguide</h1>
+  <h1><%= t('.mountain_view.main.styleguide_title') %></h1>
 </div>
 
 <% if mv_components.any? %>
   <div class="mv-main__description">
-    Select one of the components from the side to view its examples and documentation.
+    <%= t('mountain_view.index.description') %>
   </div>
 <% else %>
   <div class="mv-hint">
-    <h2><span>Hint:</span>You haven't created any components yet</h2>
+    <h2><%= t('mountain_view.index.hint_component_html') %></h2>
     <ul>
       <li>
-        You can generate your first component by running:
+        <%= t('mountain_view.index.tip_first_component_html') %>:
         <br>
         <pre>rails generate mountain_view:component component-name</pre>
       </li>
       <li>
-        Or manually create the following files:
+        <%= t('mountain_view.index.tip_second_component_html') %>:
```

Comment:
The `:` at the end of the line should go in the translation string (in `en.yml`)